metrics.proto

```diff
@@ -0,0 +1,83 @@
+// FROM https://github.com/prometheus/client_model/blob/proto3/metrics.proto
```

Comment:
This isn't merged, do we want it instead of what's in master? https://github.com/prometheus/client_model/pull/14